test/pdfs/issue16038.pdf.link

```diff
@@ -0,0 +1 @@
+https://github.com/mozilla/pdf.js/files/10703411/dvisvgm-gh228.pdf
```

Comment:
This test-case looks simple enough that we ought to be able to just add the file to the repository, since that's always preferred whenever possible.